workflows/cve-fixer/.claude/commands/cve.find.md

```
@@ -54,13 +54,45 @@ Report: artifacts/cve-fixer/find/cve-issues-20260226-145018.md
+
+   ```bash
+   # Diagnose accessibility before attempting auth
+   TOKEN_SET=$([ -n "${JIRA_API_TOKEN}" ] && echo "yes" || echo "no")
+   EMAIL_SET=$([ -n "${JIRA_EMAIL}" ] && echo "yes" || echo "no")
+   echo "JIRA_API_TOKEN accessible to bash: $TOKEN_SET"
+   echo "JIRA_EMAIL accessible to bash: $EMAIL_SET"
+   ```
+
+   - If either is **"no"** → the Ambient custom env vars are not being passed to bash
+     subprocesses. Ask the user to export them explicitly in the session:
+     ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)